packages/react/src/components/koros/Koros.stories.tsx

```diff
@@ -24,6 +24,8 @@ export default {
 
 export const Basic = () => <Koros />;
 
+export const Dense = () => <Koros dense />;
+
 export const Beat = () => <Koros type="beat" />;
 
 export const Pulse = () => <Koros type="pulse" />;
```

Comment:
I think dense property should be a property in the Playground story. It would help library users.

Reply:
Hmm.. It will probably conflict with the library update pull request, so maybe we do it later?

Reply:
Could we first merge the library update?

Reply:
Or we can merge this as well, and do it afterwards. It is only couple of lines 👍